src/Microsoft.AspNetCore.Sockets.Abstractions/IConnection.cs

```diff
@@ -13,6 +13,7 @@ public interface IConnection
         Task StartAsync();
         Task SendAsync(byte[] data, CancellationToken cancellationToken);
         Task DisposeAsync();
+        Task AbortAsync(Exception ex);
```

Comment:
This is a sign we should unify the client and sever connection objects. This will turn into `ConnectionContext`

Reply:
Agreed, but I don't want to block this PR on that right now since I don't need to.

Reply:
https://github.com/aspnet/SignalR/issues/1177 to track that